options/m_option.h

```diff
@@ -215,7 +215,7 @@ struct m_sub_options {
     const void *defaults;
     // Change flags passed to mp_option_change_callback() if any option that is
     // directly or indirectly part of this group is changed.
-    int change_flags;
+    uint64_t change_flags;
```

Comment:
This is breaking change for `option_change_callback`

Reply:
I guess that can stay int until UPDATE constants actually reach << 32.

Reply:
you can just change it, it's not a public API